graphdatascience/graph_data_science.py

```diff
@@ -173,9 +188,8 @@ def run_cypher(
         database: str
             the database on which to run the query
 
-        Returns
-        -------
-        The query result as a DataFrame
+        Returns:
+            The query result as a DataFrame
```

Comment:
NIT - this lead before to an odd rendering of `Return type: [...]`. Now it matches with the rest to just show `Returns:`